modules/oxide/gulpfile.js

```diff
@@ -89,12 +89,25 @@ gulp.task('less', function() {
 // Minify CSS
 //
 gulp.task('minifyCss', function() {
+  return gulp.src(['./build/skins/**/*.css', '!**/*.min.css'])
+    .pipe(sourcemaps.init())
+    .pipe(cleanCSS({ rebase: false }))
+    .pipe(rename({ extname: '.min.css' }))
+    .pipe(sourcemaps.write('.'))
+    .pipe(gulp.dest('./build/skins'))
+    .pipe(connect.reload());
+});
+
+//
+// Generate JS
+//
+gulp.task('generateJs', function() {
```

Comment:
Why are we adding this to make it compatible with develop? With this won't we generate the css twice and bundling it all up? Seems like it'd take more time and space.

Reply:
No we generate them only once